index.js

```diff
@@ -2,6 +2,19 @@
 
 var plugins = require('./plugins.json');
 
+for (const pluginName of Object.keys(plugins)) {
```

Comment:
I _think_ this usage of `const` is okay in Node.js v4? Might be safer to just use `var`.